### DISCUSSION TITLE
Currently trying phoniebox version 3 on a Rasperry Pi 1 - wonder if this will work!

### DISCUSSION THREAD

**Discussion by 000panther** [MiczFlor/RPi-Jukebox-RFID#1883] [Show and tell]
I am really interested in if this will still work. Currently sticking to the default Audio Jack, will keep you posted!

**Comment by wastlmuc** [comment]
Working fine so far on mine. Had to adapt nigix conf as for disabled ipv6 (I guess).
Slightly slower startup experience though.
Might this be related to local systemd services?
